src/server.cpp

```diff
@@ -1212,6 +1212,7 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
 
         const int maxPanDelay = MAX_DELAY_PANNING_SAMPLES;
```

Comment:
Do we know why this is necessary? Maybe for readability?
Might be another opportunity to check if the compiler optimizes this out or if it would be useful to just use the define in the code below.